src/logger.js

```diff
@@ -1,14 +1,17 @@
+import color_message from './color'
```

Comment:
Does this need to be it's own separate module? I feel like the logger may be able to handle the color with a simple helper function that could check if color is enabled and supported.